docs/DafnyRef/Topics.md

```diff
@@ -523,31 +521,37 @@ the recursive calls in the definition [the EvenNat equation](#eq-EvenNat) to try
 terminate.  However, there are useful ways to establish that an extreme predicate holds
 and there are ways to make use of one once it has been established.
 
-For any $\mathcal{F}$ as in [the general equation](#eq-general), I define two infinite series of well-founded
+For any $\mathcal{F}$ as in [the general equation](#eq-general), define two infinite series of well-founded
 functions, ${ {}^{\flat}\kern-1mm f}_k$ and ${ {}^{\sharp}\kern-1mm f}_k$
 where $k$ ranges over the natural numbers:
 
-<p style="text-align: center;" id="eq-least-approx" title="the least approx definition">$$
+<p style="text-align: center;" id="eq-least-approx" title="the least approx definition">
+$$
    { {}^{\flat}\kern-1mm f}_k(x) = \left\{
     \begin{array}{ll}
       \mathit{false}         & \textrm{if } k = 0 \\
       \mathcal{F}({ {}^{\flat}\kern-1mm f}_{k-1})(x) & \textrm{if } k > 0
     \end{array}
-     \right\} $$.</p>
+     \right\} 
+$$
+</p>
 
-<p style="text-align: center;" id="eq-greatest-approx" title="the greatest approx definition">$$
+<p style="text-align: center;" id="eq-greatest-approx" title="the greatest approx definition">
+$$
    { {}^{\sharp}\kern-1mm f}_k(x) = \left\{
     \begin{array}{ll}
       \mathit{true}          & \textrm{if } k = 0 \\
       \mathcal{F}({ {}^{\sharp}\kern-1mm f}_{k-1})(x) & \textrm{if } k > 0
     \end{array}
-    \right\} $$.</p>
+    \right\} 
+$$
+</p>
 
-These functions are called the _iterates_ of $f$, and I will also refer to them
+These functions are called the _iterates_ of $f$, and we will also refer to them
 as the _prefix predicates_ of $f$ (or the _prefix predicate_ of $f$, if we think
 of $k$ as being a parameter).
 Alternatively, we can define ${ {}^{\flat}\kern-1mm f}_k$ and ${ {}^{\sharp}\kern-1mm f}_k$ without mentioning $x$:
-Let $\bot$ denote the function that always returns `false`, let $\top$
+let $\bot$ denote the function that always returns `false`, let $\top$
 denote the function that always returns `true`, and let a superscript on $\mathcal{F}$ denote
 exponentiation (for example, $\mathcal{F}^0(f) = f$ and $\mathcal{F}^2(f) = \mathcal{F}(\mathcal{F}(f))$).
```

Comment:
For some reason, the second of these math expressions doesn't get rendered as math.

Reply:
It does in the GitHub preview. How are you viewing it?

Reply:
cf. https://davidcok.github.io/dafny/DafnyRef/DafnyRef#eq-least-is-exists

Reply:
Seems that the GitHub "rich diff" view doesn't render them correctly, but the preview you linked does. Thanks.